lib/dotenv/parser.rb

```diff
@@ -2,7 +2,7 @@
 require "dotenv/substitutions/command" if RUBY_VERSION > "1.8.7"
 
 module Dotenv
-  class FormatError < SyntaxError; end
+  class FormatError < RuntimeError; end
```

Comment:
Is this causing issues, or what is the motivation to switch from SyntaxError to RuntimeError?